images/nginx/README.md

```diff
@@ -13,12 +13,12 @@
 
 A minimal nginx base image rebuilt every night from source.
 
-## Upcoming Breaking Changes
+## Breaking Changes
 
-On May 3 2023 the Chainguard nginx image will be rebuilt with several improvements, including
-breaking changes. You may need to take action to update your application to prevent disruption. 
+On May 3 2023 the Chainguard nginx image was rebuilt with several improvements, including
```

Comment:
Can you link an issue/PR to describe why this change was made, or what kinds of changes were made?